docs/security/security-best-practices.md

```diff
@@ -4,37 +4,67 @@ menu_title: Security best practices
 tags: reference
 ---
 
-## Introduction
+## Introduction to Security Practices
 
-This guide covers the best practices for securing WooCommerce stores, including hardening WordPress, keeping plugins and themes up to date, implementing secure coding practices, and protecting user data. By following these recommendations, developers can build secure and resilient WooCommerce stores that protect both their business and their customers.
+Security is extremely important for online stores. A breach can lead to significant financial loss, reputational damage, and erosion of customer trust. Implementing robust security practices is crucial. This guide covers the best practices for securing WooCommerce stores, including hardening WordPress, keeping plugins and themes up to date, implementing secure coding practices, and protecting user data. By following these recommendations, developers can build secure and resilient WooCommerce stores that protect both their business and their customers.
 
-## Audience
+### Audience
 
 This guide is intended for developers who are familiar with WordPress and WooCommerce and want to improve the security of their online stores.
 
-## Prerequisites
+### Prerequisites
 
 To follow this guide, you should have:
 
 1. A basic understanding of WordPress and WooCommerce.
 2. Access to a WordPress website with WooCommerce installed and activated.
 
-## Step 1 - Keep WordPress, WooCommerce, and plugins up to date
+## Security Standards
+
+We recommend that you maintain the following standards: 
+
+- **PHPCS**: The PHP Code Sniffer tool helps ensure your code adheres to specific coding standards, which can prevent common security issues. Extensions have no security issues when testing against PHPCS rules.
+
+- **SemGrep**: Focuses on identifying patterns that may indicate security vulnerabilities, allowing developers to write more secure code. Extensions have no security issues when testing against SemGrep rules.
+
+Understanding and implementing these standards can significantly reduce the risk of security vulnerabilities in your extensions and themes.
+
+### Common Security Pitfalls and How to Avoid Them
+
+Several common security pitfalls can easily be avoided with vigilance and best practices:
+
+- **SQL Injection**: Ensure all database queries are properly sanitized.
+- **Cross-Site Scripting**: Sanitize all user input to prevent malicious code from being executed.
+- **File Inclusions**: Limit file inclusion to prevent unauthorized access to the file system.
+
+Awareness and prevention of these common issues are crucial steps in securing your WooCommerce store.
+
+## Manual Testing Guidelines
+
+Even without access to automated tools like QIT, manual testing remains a vital component of securing your WooCommerce store. Here are some guidelines on how developers can manually test their extensions for security vulnerabilities:
```

Comment:
Could be worth linking to QIT and spelling it out since folks may not know it:

```suggestion
Even without access to automated tools like the [Woo Quality Insights Toolkit (QIT)](https://qit.woo.com), manual testing remains a vital component of securing your WooCommerce store. Here are some guidelines on how developers can manually test their extensions for security vulnerabilities:
```